vllm/entrypoints/llm.py

```diff
@@ -1215,11 +1215,18 @@ def sleep(self, level: int = 1):
         self.reset_prefix_cache()
         self.llm_engine.sleep(level=level)
 
-    def wake_up(self):
+    def wake_up(self, tags: Optional[list[str]] = None):
         """
         Wake up the engine from sleep mode. See the :meth:`sleep` method
-        for more details."""
-        self.llm_engine.wake_up()
+        for more details.
+        
+        :param tags: An optional list of tags to reallocate the engine memory 
```

Comment:
Plz follow the docstring format used in vLLM, like the following
```suggestion
        Args:
            tags: An optional list of tags to reallocate the engine memory 
```

Reply:
fixed!